golang/cosmos/x/swingset/types/msgs.go

```diff
@@ -104,6 +104,28 @@ func (msg MsgWalletAction) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{msg.Owner}
 }
 
+// GetSignBytes encodes the message for signing
+func (msg MsgWalletAction) GetSignBytes() []byte {
+	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&msg))
+}
+
+// GetSignBytes encodes the message for signing
+func (msg MsgWalletSpendAction) GetSignBytes() []byte {
```

Comment:
Please move the `MsgWalletSpendAction` methods down to the end of the ...SpendAction section below.